docs/examples/spectrum/average_photon_energy.py

```diff
@@ -0,0 +1,198 @@
+"""
+Average Photon Energy Calculation
+============================
+
+Calculation of the Average Photon Energy from SPECTRL2 output.
+"""
+
+# %%
+# Introduction
+# ------------
+# This example demonstrates how to use the
+# :py:func:`~pvlib.spectrum.average_photon_energy` function to calculate the
+# Average Photon Energy (APE, :math:`\overline{E_\gamma}`) of spectral
+# irradiance distributions simulated using :py:func:`~pvlib.spectrum.spectrl2`.
+# More information on the SPECTRL2 model can be found in [1]_.
+# The APE parameter is a useful indicator of the overall shape of the solar
+# spectrum [2]_. Higher (lower) APE values indicate a blue (red) shift in the
+# spectrum and is one of a variety of such characterisation methods that is
+# used in the PV performance literature [3]_.
+#
+# To demonstrate this functionality, first we need to simulate some spectra
```

Comment:
```suggestion
# To demonstrate this functionality, first we will simulate some spectra
```

Just to avoid any implication that the method is somehow specific to SPECTRL2

Reply:
Good point. I have implemented both suggested wording revisions for this